### PR TITLE
Feature/wa 284 move code to module

### PR DESCRIPTION
**Pull request by DmitryBespalov**
- Moved non-UI files to corresponding modules

This pretty much changes everything

### CODE REVIEW

File: Common/CommonTests/Info.plist [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>BNDL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.2.0</string>
+	<string>1.0</string>
```
Comment (sche) [inline Common/CommonTests/Info.plist:18]: should it be like this?

File: safeUIKit/safeUIKit/RuleLabel.swift [hunk 2]
```diff
@@ -40,7 +40,7 @@ final class RuleLabel: UILabel {
     }
 
     private func configure() {
-        font = FontName.body // TODO: 23/02/18 support dynamic type change observing
```
Comment (sche) [inline safeUIKit/safeUIKit/RuleLabel.swift:43]: why?

File: Common/CommonTests/Info.plist (hunk 1 above)
Comment (DmitryBespalov) [reply Common/CommonTests/Info.plist:18]: auto-changed by Xcode, no bother - it'll be changed by agvtool later

File: safeUIKit/safeUIKit/RuleLabel.swift (hunk 2 above)
Comment (DmitryBespalov) [reply safeUIKit/safeUIKit/RuleLabel.swift:43]: since we have a ticket for that

**Review by sche — APPROVED** [review]

**Pull request merged**